### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by AmaduKamara**
# Query multiple tables
Completed all multiple queries on tables

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -14,3 +11,61 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+
+-- Who owns the most animals?
+SELECT owners.full_name, COUNT(animals.name) AS total FROM owners LEFT JOIN animals ON animals.owner_id=owners.id 
+GROUP BY owners.full_name ORDER BY total DESC LIMIT 1;
+
+-- NOT PART OF THE REQUIREMENT - FOR REFERENCE
+-- Returns all owners and animals total they own
+SELECT owners.full_name, COUNT(animals.name) AS total FROM owners LEFT JOIN animals ON animals.owner_id=owners.id 
+GROUP BY owners.full_name;
```
Comment (DammyShittu) [inline data.sql]: Queries are meant to be in the `queries.sql`. Kindly move the above queries into the appropriate file. 😉

File: data.sql [hunk 2]
```diff
@@ -14,3 +11,61 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
+/* Insert data into the species table */
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
+
+/* Modify inserted animals so it includes the species_id value */
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
```
Comment (DammyShittu) [inline data.sql:29]: [OPTIONAL]

- Great implementation! 👍
Here 👇 is another way you can `UPDATE` the `animals` table

```
 UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @AmaduKamara, 

Great job so far!

There are some issues that you still need to work on to go to the next milestone but you are almost there!


## Required Changes ♻️

_Check the comments under the review._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (AmaduKamara) [reply data.sql]: Oooh, yeah. Sorry, bro.  
Got it.

File: data.sql (hunk 2 above)
Comment (AmaduKamara) [reply data.sql:29]: Okay, bro. Thanks for sharing that. 🤝

**Review by mmsesay — APPROVED** [review]
Hi @AmaduKamara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**